src/go/rpk/pkg/cli/cmd/redpanda/start.go

```diff
@@ -579,7 +576,7 @@ func buildRedpandaFlags(
 		// If --io-properties-file and --io-properties weren't set, try
 		// finding an IO props file in the default location.
 		sFlags.ioPropertiesFile = rp.GetIOConfigPath(
-			filepath.Dir(conf.ConfigFile),
+			filepath.Dir(conf.FileLocation()),
```

Comment:
What happens if there was no file here? I think in the old code, this would be the default path.

Reply:
i c, this is now set to the default config path if not present